cloudhsmtool/cloudhsm.log

```diff
@@ -0,0 +1 @@
+bash: /opt/cloudhsm/bin/cloudhsm_client: No such file or directory
```

Comment:
Does this nee commiting?

Reply:
No, I don't suppose it does really. Not sure how that slipped through.